src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/ResourceHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation
```

Comment:
Why is it called `ResourceHelper`? Maybe a better name is `WindowsSetting`?

Reply:
Because it read the data from json. We should not rename it to WindowsSettingHelper.

Maybe `WindowsSettingsJsonHelper` would be good.

Reply:
And `WindowsSettings` is used already

Reply:
Changed to `JsonSettingsListHelper`